tools/rerun/run.py

```diff
@@ -149,9 +161,8 @@ def load_data(self):
   parser.add_argument("--ecam", action="store_true", help="Show wide camera")
   parser.add_argument("--dcam", action="store_true", help="Show driver monitoring camera")
   parser.add_argument("--print_services", action="store_true", help="List out openpilot services")
-  parser.add_argument("--services", default=[], nargs='*', help="Specify openpilot services that will be logged.\
-                                                                No service will be logged if not specified.\
-                                                                To log all services include 'all' as one of your services")
+  parser.add_argument("--services", default=["all"], nargs='*', help="Specify openpilot services that will be logged.\
```

Comment:
is rerun still too slow to remove this?

Reply:
no, we can remove it now. I was just putting it there too speed up some testing. That will be removed at the end